### ISSUE TITLE
'str' object has no attribute 'get'

### ISSUE THREAD

**Issue by fahdmirza** [jabberjabberjabber/llmscripts#1]
Hi,
 I tried it out in Ubuntu , and got following error.

air) Ubuntu@0136-ict-prxmx50056:~/llmscripts$ python3 llm-utility.py "/home/Ubuntu/testimages" --api-url http://localhost:5001
Processing file 1 of 1: /home/Ubuntu/testimages/4a.jpgError processing task 'metadata': unsupported operand type(s) for +: 'NoneType' and 'int'

Error processing /home/Ubuntu/testimages/4a.jpg: 'str' object has no attribute 'get'

Thanks.

**Comment by jabberjabberjabber** [comment]
Yeah, that happens when the LLM returns something that can't be parsed. Not much you can do except to force it to use a grammar file which slows it down like 50%.

**Comment by fahdmirza** [comment]
I was looking to review this tool for my channel at https://www.youtube.com/@fahdmirza and won't look good with this error. I have tried various their images but same error. Do you have any working example or workaround this? Thanks.

**Comment by jabberjabberjabber** [comment]
Dude please don't review it. It is not good and I suck at coding I made it for myself and it is a work in progress.

**Comment by fahdmirza** [comment]
No Problem, thanks. But good effort nonetheless.
